frontend/src/components/helpers/auth/PersistentLogin.js

```diff
@@ -7,14 +7,31 @@ import { SocketMessages } from "../socket-messages/SocketMessages";
 import { GenericLoader } from "../../generic-loader/GenericLoader";
 import { PromptRun } from "../../custom-tools/prompt-card/PromptRun";
 
+let selectedProductStore;
+let selectedProduct;
+try {
+  selectedProductStore = require("../../../plugins/llm-whisperer/store/select-product-store.js");
+} catch {
+  // Ignore if hook not available
+}
```

Comment:
@jagadeeswaran-zipstack Here's a straightforward approach to read the `useSelectedProductStore`,

```suggestion
let useSelectedProductStore;
try {
  useSelectedProductStore = require("../../../plugins/llm-whisperer/store/select-product-store.js")
    .useSelectedProductStore;
} catch {
  // Ignore if hook not available
}
```

Reply:
@tahierhussain This was wrote this way because you shouldn't call hooks conditionally as per sonar config.
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=800&id=Zipstack_unstract&open=AZKZdv-sy9qRwus5zxKC